entity/person.go

```diff
@@ -0,0 +1,9 @@
+package entity
+
+type (
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Also, you are only declaring only one struct here, do it without `()`
```go
// Person -  Description
type Person struct {
	ID        string `json:"id"`
	FirstName string `json:"first_name"`
	LastName  string `json:"last_name"`
}

```